pkgs/by-name/no/nolibc/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenvNoLibc,
+  linuxHeaders,
+}:
+
+stdenvNoLibc.mkDerivation (finalAttrs: {
+  pname = "nolibc";
+
+  inherit (linuxHeaders) version src;
+
+  # Nolibc depends on these headers
+  propagatedBuildInputs = [ linuxHeaders ];
+
+  setSourceRoot = ''
+    sourceRoot=$(echo */tools/include/nolibc)
```

Comment:
Can you explain what is the sourceRoot selected with globbing?

Reply:
It's already documented in the stdenv docs https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#setsourceroot-var-stdenv-setsourceroot